bundles/edu.kit.kastel.sdq.eclipse.grading.core/src/edu/kit/kastel/sdq/eclipse/grading/core/ArtemisController.java

```diff
@@ -346,4 +448,126 @@ public void startAssessment(ISubmission submissionID) {
 		}
 	}
 
+	@Override
+	public boolean submitSolution(ICourse course, IExercise exercise, IProjectFileNamingStrategy projectNaming) {
+		final File eclipseWorkspaceRoot = ResourcesPlugin.getWorkspace().getRoot().getLocation().toFile();
+		File exeriseRepo = projectNaming.getProjectFileInWorkspace(eclipseWorkspaceRoot, exercise, null);
+		File gitFileInRepo = projectNaming.getGitFileInProjectDirectory(exeriseRepo);
+		try {
+			GitHandler.commitExercise(username, username + "@student.kit.edu", "Test Commit Artemis", gitFileInRepo);

```

Comment:
Please do not brand this to kit. Simply use the mail of the login (user) object. In addition, you may provide a more detailed commit message.